writers/cassandra/messages.go

```diff
@@ -0,0 +1,31 @@
+package cassandra
```

Comment:
Rename the file (and corresponding test file) to be consistent with other writers.

Reply:
I would suggest to rename files in other writers like this.

Reply:
Hm... I agree with @anovakovic01 make more sense for naming. But not sure if should be effort now.